src/vue/components/morph-parts/definitions-list.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <div class="alpheios-morph__definition_list">
+
+        <div class="alpheios-popup__definitions--placeholder"
+            v-show="! $store.getters['app/shortDefDataReady']"
+        >
+            <progress-bar :text="l10n.getText('PLACEHOLDER_LEX_DATA_LOADING')" />
+        </div>
+
+        <div :data-lemmakey="lexeme.lemma.ID"  class="alpheios-morph__definition" 
+            v-for="(definition, dindex) in definitions" :key="definition.ID"
+            v-show="$store.getters['app/shortDefDataReady']"
+        >
+
+            <span class="definition_index" v-if="definitions.length > 1">{{ definitionIndex(dindex) }}</span>
+            <shortdef :definition="definition"></shortdef>
+
+        </div>
+    </div>
+</template>
+<script>
+  import { Definition } from 'alpheios-data-models'
+  import ProgressBar from '@/vue/components/progress-bar.vue'
+  import ShortDef from '@/vue/components/shortdef.vue'
+
+  export default {
+    name: 'DefinitionsList',
+    components: {
+        progressBar: ProgressBar,
+        shortdef: ShortDef
+    },
+    inject: ['app', 'l10n'],
+    storeModules: ['app'],
+    props: {
+      lexeme: {
+        type: Object,
+        required: true
+      }
+    },
+    computed: {
+      definitions () {
+        let definitionsLocal = []
+        if (this.$store.getters['app/shortDefDataReady'] && this.lexeme.meaning && this.lexeme.meaning.shortDefs && this.lexeme.meaning.shortDefs.length > 0) {
+          definitionsLocal = this.lexeme.meaning.shortDefs
+        // We don't need the deduplication code below as of now; it is here just for historic reference
+        /* for (const def of lexeme.meaning.shortDefs) {
+            // for now, to avoid duplicate showing of the provider we create a new unproxied definitions
+            // object without a provider if it has the same provider as the morphology info
+            if (def.provider && lexeme.provider && def.provider.uri === lexeme.provider.uri) {
+              definitions.push(new Definition(def.text, def.language, def.format, def.lemmaText))
+            } else {
+              definitions.push(def)
+            }
+          } */

```

Comment:
I'm not sure when this code was put in and why it is not needed now, but probably we should not use comments to preserve it in the history. It's probably better to store it somewhere else or refer to the commit history where this code was added. What do you think?

Reply:
agree we could take this out.

Reply:
done